QueryDemo.ipynb

```diff
@@ -0,0 +1,3670 @@
+       " ('2MASS J04291842-3123568', 67.326773, -31.399113, None, None, '0429-3123', 'Cruz03', None, None),\n",
+       " ('WISE J144806.48-253420.3', 222.027, -25.5724, None, '2000', None, 'Thom13', None, None),\n",
+       " ('2MASS J13080663-4925505', 197.0275, -49.4307, None, '2000', None, 'Folk12', None, None),\n",
+       " ('WISEA J211543.59-322540.4', 318.9312, -32.4273, None, '2000', None, 'Kirk14', None, None),\n",
+       " ('ULAS J222711-004547', 336.7956, -0.7632, None, '2000', None, 'Maro14', None, None),\n",
+       " ('2MASS J11181292-0856106', 169.553849, -8.936299, None, None, '1118-0856', 'Kirk10', None, None),\n",
+       " ('2MASS J15461461+4932114', 236.5611, 49.5362, None, '2000', None, 'Metc08', None, None),\n",
+       " ('CWISE J133427.70-273053.1', 203.6154266, -27.5147656, 2015.4041, 'ICRS', None, 'Roth', None, None),\n",
+       " ('2MASS J16265698+3954482', 246.73745, 39.913391, None, None, None, 'Cruz03', None, None),\n",
+       " ('2MASS J20140359-2016217', 303.5152, -20.2727, None, '2000', None, 'Cruz03', None, None),\n",
+       " ('ULAS J131106.96-013742.3', 197.7789, -1.6284, None, '2000', None, 'Maro15', None, None),\n",
+       " ('CWISEP J235547.99+380438.9', 358.9499583333333, 38.07747222222223, None, None, None, 'Meis20.74', None, None),\n",
+       " ('2MASS J00550470-3052005', 13.769167, -30.866667, None, None, None, 'Cruz07', None, None),\n",
+       " ('DENIS-P J0644143-284141', 101.0601, -28.6949, None, '2000', None, 'Phan08', None, None),\n",
+       " ('2MASS J01174748-3403258', 19.447833, -34.057167, None, None, '0117-3403', 'Cruz03', None, None),\n",
+       " ('2MASS J17195973-0640313', 259.99887, -6.6753611, None, None, None, 'Reid08.1290', None, None),\n",
+       " ('PSO J136.5380-14.3267', 136.5389, -14.3272, None, '2000', None, 'Best20.257', None, None),\n",
+       " ('51 Eri b', 69.40041, -2.47361, None, None, '0437-0228b', 'Maci15', None, None),\n",
+       " ('2MASS J21144103-4339531', 318.6713, -43.6652, None, '2000', None, 'Gagn15.33', None, None),\n",
+       " ('2MASS J12321827-0951502', 188.0761245, -9.863944, None, None, '1232-0951', 'Reid08.1290', None, None),\n",
+       " ('WISEA J011154.36-505343.2', 17.9765, -50.8954, None, '2000', None, 'Luhm14.126', None, None),\n",
+       " ('ULAS J133836.97-022910.7', 204.6543, -2.4854, None, '2000', None, 'Zhan17.3040', None, None),\n",
+       " ('2MASSI J1658037+702701', 254.51585415, 70.45043333, None, None, '1658+7026', 'Gizi00', None, None),\n",
+       " ('CFBDS J022644.65-062522.0', 36.686, -6.4228, None, '2000', None, 'Albe11', None, None),\n",
+       " ('2MASS J05574102-1333264', 89.420954, -13.557334, None, None, None, 'Cruz18', None, 'not M'),\n",
+       " ('2MASS J09393548-2448279', 144.897834, -24.80775, None, None, '0939-2448', 'Tinn05.2326', None, None),\n",
+       " ('2MASS J22590210+1156035', 344.758944, 11.933929, None, None, None, 'Zhan18.2054', None, 'Discovery reference from t
```

Comment:
I would not have this cell. You've shown how to use the Astrodbkit query methods and how to use sql_query, but now you're going into low-level API for SQLAlchemy itself. While useful, it probably doesn't belong in a notebook demonstrating quick/useful queries of SIMPLE.